hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/MapCodecTemplate.java

```diff
@@ -719,4 +719,19 @@ Object addEntryListenerWithPredicate(String name, Data predicate, boolean includ
 
     @Request(id = 59, retryable = false, response = ResponseMessageConst.VOID)
     void clearNearCache(String name, Address target);
+
+    /**
+     * Provides functionality to iterate over entries of a partition.
+     * The ordering of iteration over entries is undefined.
+     * java.util.Iterator#next() may return null if the entry is no longer present, has expired or has been evicted.
```

Comment:
Can you explain java.util.Iterator#next() without any java dependency? In the protocol document, we try not to be java specific. You can explain in terms of MapKeyIteratorResult since it shall go into the document as a new type. But what about "next" method, there is no such method for the protocol but the list is only converted to be an array of Data in the protocol document. So, may be we do not use next method name at all, you can say the next item in the array.